packages/mux-player/src/media-theme-mux.html

```diff
@@ -0,0 +1,641 @@
+    audio="{{audio}}"
+    class="size-{{playerSize}}"
+    exportparts="layer, media-layer, poster-layer, vertical-layer, centered-layer, gesture-layer"
+  >
+    <slot name="media" slot="media"></slot>
+    <media-poster-image
+      slot="poster"
+      part="poster"
+      src="{{poster}}"
+      placeholder-src="{{placeholder}}"
+    ></media-poster-image>
+    <media-loading-indicator slot="centered-chrome" no-auto-hide></media-loading-indicator>
+
+    <template partial="TitleDisplay">
+      <template if="title">
+        <media-text-display part="top title display" class="title-display"> {{title}} </media-text-display>
+      </template>
+    </template>
+
+    <template partial="PlayButton">
+      <media-play-button
+        part="{{section ?? 'bottom'}} play button"
+        disabled="{{disabled}}"
+        aria-disabled="{{disabled}}"
+      >
+        <svg aria-hidden="true" viewBox="0 0 24 24" slot="play">
+          <path d="m6.73 20.93 14.05-8.54a.46.46 0 0 0 0-.78L6.73 3.07a.48.48 0 0 0-.73.39v17.07a.48.48 0 0 0 .73.4Z" />
+        </svg>
+        <svg aria-hidden="true" viewBox="0 0 24 24" slot="pause">
+          <path
+            d="M6 19.5a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-15a.5.5 0 0 0-.5-.5h-3a.5.5 0 0 0-.5.5v15ZM14.5 4a.5.5 0 0 0-.5.5v15a.5.5 0 0 0 .5.5h3a.5.5 0 0 0 .5-.5v-15a.5.5 0 0 0-.5-.5h-3Z"
+          />
+        </svg>
+      </media-play-button>
+    </template>
+
+    <template partial="SeekBackwardButton">
+      <media-seek-backward-button
+        seek-offset="{{backwardSeekOffset}}"
+        part="{{section ?? 'bottom'}} seek-backward button"
+        disabled="{{disabled}}"
+        aria-disabled="{{disabled}}"
+      >
+        <svg aria-hidden="true" viewBox="0 0 22 24" slot="backward">
+          <path d="M11 6V3L5.37 7 11 10.94V8a5.54 5.54 0 0 1 1.9 10.48v2.12A7.5 7.5 0 0 0 11 6Z" />
+          <text class="value" transform="translate(2.5 21)" style="font-size: 8px; font-family: 'ArialMT', 'Arial'">
+            {{backwardSeekOffset}}
+          </text>
+        </svg>
+      </media-seek-backward-button>
+    </template>
+
+    <template partial="SeekForwardButton">
+      <media-seek-forward-button
+        seek-offset="{{forwardSeekOffset}}"
+        part="{{section ?? 'bottom'}} seek-forward button"
+        disabled="{{disabled}}"
+        aria-disabled="{{disabled}}"
+      >
+        <svg aria-hidden="true" viewBox="0 0 22 24" slot="forward">
+          <path d="M11 6V3l5.61 4L11 10.94V8a5.54 5.54 0 0 0-1.9 10.48v2.12A7.5 7.5 0 0 1 11 6Z" />
+          <text class="value" transform="translate(10 21)" style="font-size: 8px; font-family: 'ArialMT', 'Arial'">
+            {{forwardSeekOffset}}
+          </text>
+        </svg>
+      </media-seek-forward-button>
+    </template>
+
+    <template partial="MuteButton">
+      <media-mute-button part="bottom mute button" disabled="{{disabled}}" aria-disabled="{{disabled}}">
+        <svg aria-hidden="true" viewBox="0 0 24 24" slot="high">
```

Comment:
suggestion(non-blocking): It would be unfortunate if we lost the ability for designers to create svgs that they could just use with the src. This would be a good thing to follow up on so we don't have to again inline everything. The same applies to the styles.css change imo.